liquibase-core/src/main/java/liquibase/configuration/AbstractMapConfigurationValueProvider.java

```diff
@@ -0,0 +1,82 @@
+package liquibase.configuration;
```

Comment:
This seems like a good class for a synopsis at the top.  Is it the top-level Map for all properties?